### DISCUSSION TITLE
Donde se hacen las katas?

### DISCUSSION THREAD

**Discussion by konobabuno** [LaunchX-InnovaccionVirtual/CursoIntroPython#27] [Q&A] [answered]
Los ejercicios los están haciendo en una copia de las notebooks de Jupyter que se nos proporcionan por este medio, o se hace una notebook aparte por nuestra cuenta en donde se implementan los ejercicios? Saludos:)

**Comment by MauricioNovelo** [comment, marked as answer]
Podrías descargar el notebook y realizarlos desde ahí, o bien podrías realizar tus notebooks desde github o hacerlos en visual studio y luego subirlos

**Reply by ramon-ortega** [reply]
Yo estoy realizando mis propios archivos .ipynb. Para la kata numero 2, tome screenshot de lo del entorno virtual. Esto lo discutimos en la junta del día de ayer.

**Reply by konobabuno** [reply]
Si lo vi, pero no me quedo muy claro, muchas gracias:)

**Reply by SantQuintana** [reply]
Donde se suben los trabajos?

**Reply by PlayerOneV** [reply]
> Donde se suben los trabajos?

Aun no dicen, creo que lo aclararan el viernes.

**Reply by Fernando0409** [reply]
@SantQuintana hasta el viernes nos pasaran el formulario para pasar el link de nuestro repositorio. Por mientras puedes crear tu propio repo aqui en github e ir subiendo tus actividades ahi

**Reply by ga-mx** [reply]
> @SantQuintana hasta el viernes nos pasaran el formulario para pasar el link de nuestro repositorio. Por mientras puedes crear tu propio repo aqui en github e ir subiendo tus actividades ahi

¿Osea que si tenemos que clonar este repo en nuestras cuentas? Eso fue algo que por mas que busque, no me quedo tampoco claro.

**Reply by Fernando0409** [reply]
@ga-mx la que tú decidas. Clona esge repo o crea el tuyo en tu cuenta y lo vas organizando.

**Reply by ga-mx** [reply]
@Fernando0409 ¿entonces no dieron instrucciones especificas sobre eso, correcto? Tuve que comenzar el programa un par de días tarde y me estoy poniendo al día, pero no encuentro la respuesta a esa duda por ningun lado.

**Reply by Fernando0409** [reply]
Asi es @ga-mx , en la sesión del viernes nos dirán todo lo relacionado con la entrega de los ejercicios que seguramente será por un formulario con el link de tu repo

**Reply by GabrielVillagran** [reply]
> @Fernando0409 ¿entonces no dieron instrucciones especificas sobre eso, correcto? Tuve que comenzar el programa un par de días tarde y me estoy poniendo al día, pero no encuentro la respuesta a esa duda por ningun lado.

Asi es, por mientras como mencionaron anteriormente, puedes clonar el repositorio o crear uno tu (yo opte esta segunda pro mayor facilidad), el viernes nos diran que onda con la subida de las katas, aun hay chance para resolverlas, aqui estramos para ayudar si lo requieres lml

**Reply by ga-mx** [reply]
Perfecto, muchas gracias a ambos @Fernando0409, @GabrielVillagran. Estaba justamente en la repetición del Instragram Live y veo que Fernanda lo menciona al minuto 11:00; dato por si a alguien más le sirve.

**Reply by EdgarValenciaR** [reply]
@GabrielVillagran Has resuelto mi duda a pesar de que el viernes estuve en la sesión!

**Reply by RodolfoMorquecho** [reply]
Hola amigos, hice todas mis katas a nivel local pero veo que muchos dicen que se tienen que poner en el repositorio y hacer fork, a caso debia de ir subiendo progreso de cada kata aca en github? alguien ilumineme, entre a todos los lives y habia entendido que no era necesario lo de los forks

**Reply by theblower0** [reply]
> 

Hola Rodolfo, tienes que crear tu repo en tu cuenta y ahí subir tus katas, no es necesario crear ni forks ni nada más
